py3status/modules/bitcoin_price.py

```diff
@@ -65,6 +56,7 @@ class Py3status:
     format_separator = ', '
     hide_on_error = False
     markets = 'btceUSD, btcdeEUR'
+    request_timeout = 5
```

Comment:
to me `5` feels a bit short how about `10`?

Although I'd be inclined to just remove this option as we should really do this sort of thing universally for all modules that make requests.  I'd also like to add other universal options for requests including things like proxies (although `requests` would be needed for that as they are too complex for us to be supporting ourselves.

Reply:
`10` and option now hardcoded / concealed. 👍